e2e-tests/portal/cypress/e2e/journeys/login/resend-sign-in-link.spec.js

```diff
@@ -17,14 +18,17 @@ context('Resending sign in links', () => {
     let bank1Maker1Id;
 
     beforeEach(() => {
-      const { username } = BANK1_MAKER1;
       cy.getUserByUsername(username).then(({ _id }) => {
         bank1Maker1Id = _id;
       });
       cy.resetPortalUserStatusAndNumberOfSignInLinks(username);
       cy.enterUsernameAndPassword(BANK1_MAKER1);
     });
 
+    after(() => {
+      cy.resetPortalUserStatusAndNumberOfSignInLinks(username);
+    });
```

Comment:
Cypress recommends in their best practice doc that we shouldn't really use an `after` block, and instead we should have a `before` block in `login.spec.js`: https://docs.cypress.io/guides/references/best-practices#Using-after-Or-afterEach-Hooks